src/verification/android.rs

```diff
@@ -56,13 +57,16 @@ impl Drop for Verifier {
 }
 
 impl Verifier {
+    /// Creates a new instance of a TLS certificate verifier that utilizes the
+    /// Android certificate facilities
     pub fn new() -> Self {
         Self {
             #[cfg(any(test, feature = "ffi-testing"))]
             test_only_root_ca_override: None,
         }
     }
 
+    /// Creates a test-only TLS certificate verifier which trusts our fake root CA cert.
     #[cfg(any(test, feature = "ffi-testing"))]
     pub fn new_with_fake_root(root: &[u8]) -> Self {
```

Comment:
Suggestion: I believe this can be made `pub(crate)` like the other platforms. This is a leftover from when the tests which used this were outside of `src`.

Reply:
Tacked on an additional commit to implement this change.